AudioKit/Core Classes/AKManager.h

```diff
@@ -29,6 +29,9 @@ NS_ASSUME_NONNULL_BEGIN
 /// Determines whether or not to log
 @property BOOL isLogging;
 
+/// Place to put lines of testing logs
+@property NSMutableArray *testLog;
+
```

Comment:
Well, like I said, this will be changing and going away, so I hate to put more effort into it, but if you wanna, you can.